pr_check.sh

```diff
@@ -81,3 +81,9 @@ sed -i -e 's/mas_sso/redhat_sso/g' internal/connector/internal/environments/inte
 docker login -u "${QUAY_USER}" -p "${QUAY_TOKEN}" quay.io
 docker build -t "$IMAGE_NAME" -f Dockerfile_integration_tests .
 docker run -i "$IMAGE_NAME"
+
+# ensure that the kas-fleet-manager docker image can be built when PR check runs
```

Comment:
Do we need this? in midstream the pipeline already tries to buy the image right?
Maybe let's keep this simple

Reply:
was thinking the same. in the midstream the image is built when github secrets are provided as env vars. does it mean that we will built an image twice?

Reply:
> Do we need this? in midstream the pipeline already tries to buy the image right?

The image is only built when something is merged on `main` for both Github and midstream pipelines. The usefulness of this is that we'll be sure that before we merge something in main, it can successful build the image.

Reply:
but in midstream it won't be merged until the image is built no?

Reply:
Without this, if a PR introduce a breaking step as far as image build is concerned, that PR will be merged on the midstream KFM repo, which then kicks a midstream pipeline to build the image: it is at this step where it'll fail - long after the PR pipeline has finished.

Reply:
Ok so the the image won't be pushed to the repository but the content will be merged to midstream main.
What concerns me is that that file is becoming kind of a container for tasks in midstream. It might make sense to separate in different jobs for the midstream PR CI flow.

Reply:
> Ok so the the image won't be pushed to the repository but the content will be merged to midstream main.

Yes. 

> What concerns me is that that file is becoming kind of a container for tasks in midstream. It might make sense to separate in different jobs for the midstream PR CI flow.

Yes. This is very valid concern. Do you mind creating a JIRA for this? I won't be able to get to that as part of this PR but it is something we should really revisit for both midstream and upstream pipelines as mentioned in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1641#discussion_r1138641797

Reply:
I will create a jira for it

Reply:
Thanks